daemon/Cargo.toml

```diff
@@ -32,7 +32,7 @@ tokio-util = { version = "0.6", features = ["codec"] }
 tracing = { version = "0.1" }
 tracing-subscriber = { version = "0.2", default-features = false, features = ["fmt", "ansi", "env-filter", "chrono", "tracing-log", "json"] }
 uuid = { version = "0.8", features = ["serde", "v4"] }
-xtra = { version = "0.5", features = ["with-tokio-1"] }
+xtra = { git = "https://github.com/Restioson/xtra", features = ["with-tokio-1"] }
```

Comment:
Can you use the `[patch.crates-io]` feature in the root `Cargo.toml` please?

Reply:
sure! makes sense